### ISSUE TITLE
[WEB] Make pages responsive to zoom in and out - resize proportionally

### ISSUE THREAD

**Issue by tiagocostasilva** [FightPandemics/FightPandemics#427]
Hi team, 
the objective would be to make the page zoom in and out proportionally responsive to it.

The description can be seen here.
This is 100%

![image](https://user-images.githubusercontent.com/45389561/82603801-33438f80-9bab-11ea-9423-9fc41e8295ce.png)

This 175%: 
![image](https://user-images.githubusercontent.com/45389561/82604057-96352680-9bab-11ea-8613-84381b606e26.png)

On this scenario on Chrome you can see the "And" text on the blue square comes up. It would be great the text format the same way. 
Also if you get it to 200% the text "info" from "Covid-info" goes down:
![image](https://user-images.githubusercontent.com/45389561/82604211-dac0c200-9bab-11ea-9948-2cdaf3f5664e.png)

Probably thats not something a person is doing all the time for sure, but in it seems a good practice/quality thing to have it (e.g. you can do it in linkedin page without login and you can see the proper resize and downsize of the page).  

Also when you do the zoom out you see some differences: 
![image](https://user-images.githubusercontent.com/45389561/82604643-823df480-9bac-11ea-8f88-11f81fa1cbeb.png)

**Also i noticed now that we have a scroll on the page to see "View Community Postings" text.**

Is this something we can achieve with this responsive page?

thanks ,
Tiago

**Comment by snowiesuet** [comment]
I don't think zooming in and out is part of the responsive development. If we were to accumulate this use case we'll have to spend a long time trying to develop for every screen size and zoom %.

**Comment by robinv85** [comment]
Seems to me that this is already responsive text sizing when I try to resize the viewport size

I can understand the "art direction" part of the design, similar to responsive images and how they resize across different screens, but for zooming I'm not sure if that will work as well, in this case I think maybe it can be adjusted with the container size around the title... this also depends on the type of units that are used, %, px, em / rems... etc.

**Comment by tiagocostasilva** [comment]
@snowiesuet @robinv85 , thanks for the feedback.

I understand your points of view here. Probably for MVP can be too much, but something for sure to take in consideration for future, and break down this in multiple tasks. What do you think? 

At this stage, and because we want to launch quickly i would say to try to put it better in the Home Page (fix size and images), in parallel we can do the work for other screens, just to see if there isn't a major layout issue with the zooming. 

Meanwhile to try to make even more clear i did the bellow video with linkedin.
I selected part of a text that goes up and down a litle bit when you do a zoom, but most of the page grow proportionally, the images, the text (do not jump to another paragraph).  

[linkedin.zip](https://github.com/FightPandemics/FightPandemics/files/4677068/linkedin.zip)

@robinv85, @snowiesuet Please give your thoughts on this? 
Am i asking too much for this phase and maybe for later? or is this a level of detail in design not to care about because.. why? Probably i can be missing some context also.

Thanks,
Tiago

**Comment by tiagocostasilva** [comment]
Hi guys,
so whats the verdict ? :) @robinv85 , @snowiesuet

**Comment by snowiesuet** [comment]
I think this would be good for future release but for now I don't think it's a priority, because this doesn't just affect homepage, it'll affect all pages.

**Comment by robinv85** [comment]
Yeah this is definitely not a priority right now... we use rems in lots of places so I think that would probably already be a pretty good part of making everything zoomable? But plenty of other things that are more high priority

**Comment by tiagocostasilva** [comment]
Perfectly agree with you @robinv85 and @snowiesuet on this that at this moment we want to launch this can be taken care in the future release. For sure we can just double check with other PMs, but i think there will be a common agreement on this. Thanks for your feedback ;)

**Comment by vikjoshi** [comment]
@tiagocostasilva , based on this thread, I'm moving this down the priority list for MVP-Release 1. Feel free to remove the MVP-Release 1 label altogether if you'd like.

**Comment by robinv85** [comment]
@vikjoshi yeah the tag was already removed, I've removed it from the milestone as well
this gets a bit confusing I guess... but the milestones are just not as visible as the tags as far as I can see, not sure if we can do something about that

**Comment by joshmorel** [comment]
I'm closing this - it's old, low priority and the resolution is unclear. If we get feedback from a user we can revisit.
